lua/telescope/make_entry.lua

```diff
@@ -77,7 +77,7 @@ do
 
     mt_file_entry.cwd = cwd
     mt_file_entry.display = function(entry)
-      local display, hl_group = entry.value
+      local display, hl_group = path.make_relative(entry.value, cwd), nil
```

Comment:
Just do `local dispay = path...` and do `local hl_group` before we actually need `hl_group`. Please don't ignore linting errors. Its here for a reason

Reply:
Done. Didn't mean to ignore the linter at the outset, only as a last resort if there was no other alternative